src/main/java/com/ss/lms/controller/BorrowerController.java

```diff
@@ -1,187 +1,116 @@
 package com.ss.lms.controller;
 
-import java.sql.Connection;
-import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.HashMap;
+
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.ApplicationArguments;
+
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
+
 
 import org.springframework.web.bind.annotation.RestController;
 
-import com.ss.lms.DAO.BookDAO;
-import com.ss.lms.DAO.BorrowerDAO;
-import com.ss.lms.DAO.BranchDAO;
-import com.ss.lms.DAO.CopiesDAO;
-import com.ss.lms.DAO.LoansDAO;
 import com.ss.lms.entity.Book;
 import com.ss.lms.entity.Borrower;
 import com.ss.lms.entity.Branch;
 import com.ss.lms.entity.Copies;
 import com.ss.lms.entity.Loans;
 import com.ss.lms.service.BorrowerService;
-import com.ss.lms.service.ConnectionUtil;
-
-
 
 @RestController
+@RequestMapping("borrower/")
 public class BorrowerController {
-	
+
 	@Autowired
 	BorrowerService borrowerService;
-	
-	@Autowired
-	Branch branch;
-	
-	
 
-	@RequestMapping(path = "borrower/{cardNo}/library/{branchId}/books/{bookId}:checkout", method = RequestMethod.POST, produces = {
-					MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE })
+
+	@PostMapping(path = "{cardNo}/branches/{branchId}/books/{bookId}:checkout", produces = {
```

Comment:
what was the reason for this change? is it just better pratice or does it work more efficently